### DISCUSSION TITLE
Worth joining Slack?

### DISCUSSION THREAD

**Discussion by Paktosan** [peering-manager/peering-manager#404] [Q&A] [answered]
Hi,
I have just been hired by DE-CIX to work on Peering Manager.
#337 sounds a bit like Slack will be retired in favour of GitHub discussions, is this true?
Or should I still join the Slack channel to not miss out on discussions?

**Comment by gmazoyer** [comment, marked as answer]
First of all congrats on working at DE-CIX :)

Joining Slack is not mandatory as it's only here for ephemeral discussions. It is worth joining only if you want to participate in some threads but it is most of the time quite calm. However if you want to start some more permanent discussions. here is the good place to do so.

**Reply by jamesditrapani** [reply]
Congratulations @Paktosan, would love to have a discussion to get to know you :)
